packages/datetime/src/datePicker.tsx

```diff
@@ -134,26 +159,29 @@ export class DatePicker extends AbstractPureComponent2<IDatePickerProps, IDatePi
 
         return (
             <div className={classNames(Classes.DATEPICKER, className)}>
-                <DayPicker
-                    showOutsideDays={true}
-                    locale={locale}
-                    localeUtils={localeUtils}
-                    modifiers={this.getDatePickerModifiers()}
-                    {...dayPickerProps}
-                    canChangeMonth={true}
-                    captionElement={this.renderCaption}
-                    navbarElement={this.renderNavbar}
-                    disabledDays={this.getDisabledDaysModifier()}
-                    fromMonth={minDate}
-                    month={new Date(displayYear, displayMonth)}
-                    onDayClick={this.handleDayClick}
-                    onMonthChange={this.handleMonthChange}
-                    selectedDays={this.state.value}
-                    toMonth={maxDate}
-                    renderDay={this.renderDay}
-                />
-                {this.maybeRenderTimePicker()}
-                {showActionsBar && this.renderOptionsBar()}
+                {this.maybeRenderShortcuts()}
+                <div>
```

Comment:
will this div cause issues? Adding this div and the maybeRenderShortcuts is all that changed here

Reply:
yes, it's a very slight HTML break... but relying on `>` selectors is not good practice IMO... I'm fine with this change